CHANGELOG.rst

```diff
@@ -4,6 +4,16 @@ conda-smithy Change Log
 
 .. current developments
 
+**Changed:**
+
+* Disk cleanup on Azure feedstocks can now be more finely controlled using keys `docker`, `cache`, and `apt` to enumerate which cleanup steps are desired. Pruning the Docker cache now must be explicitly added by enumerating cleanup steps. Setting `azure.free_disk_space = True` now only includes `cache` and `apt` steps.
+
```

Comment:
Please see `news/TEMPLATE.rst`.

Reply:
Good catch. Went ahead and removed this and added a note to a `news` file on Daniel's behalf